frontend/src/components/ConfirmationModal.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react'
+import Button from './Button'
+import styled from 'styled-components'
+import { useHistory } from 'react-router-dom'
+import { PATHS } from '../routes/Routes'
+import { JsxElement } from 'typescript'
+
+const OverlayContainer = styled.div<{ overlayBackgroundColor: string }>`
+  position: fixed;
+  min-height: 100vh;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: ${(props) => (props.overlayBackgroundColor ? props.overlayBackgroundColor : 'rgba(0, 0, 0, 0.3)')};
+  z-index: 999;
+`
+
+const MainContainer = styled.div`
+  position: fixed;
+  background-color: #fff;
+  width: 90vw;
```

Comment:
might want to add a max width so modal does not get too big

Reply:
Resolved in commit 509ab25176f49a4a836ede036eb3559e1af54833.

Reply:
max width should be an absolute value, if not width is not restricted